evalml/automl/automl_search.py

```diff
@@ -701,7 +702,7 @@ def _post_evaluation_callback(self, pipeline, evaluation_results, job_log):
                                                                           self._baseline_cv_scores.get(obj_name, np.nan))
             percent_better_than_baseline[obj_name] = percent_better
 
-        high_variance_cv = self._check_for_high_variance(pipeline, cv_scores)
+        high_variance_cv = self._check_for_high_variance(pipeline, cv_score, cv_sd)
```

Comment:
Figured it was easier to only compute the mean and std dev before `_check_for_high_variance` since we were already computing the mean outside of `_check_for_high_variance`